src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/LightBulbHelper.cs

```diff
@@ -90,7 +90,7 @@ public static async Task<IEnumerable<SuggestedActionSet>> WaitForItemsAsync(Test
             catch (OperationCanceledException) when (!cancellationToken.IsCancellationRequested)
             {
                 var version = await testServices.Shell.GetVersionAsync(cancellationToken);
-                if (Version.Parse("17.2.32314.265") >= version)
+                if (Version.Parse("17.2.32427.441") >= version)
```

Comment:
Since the fix won't be in a 17.2 release, would it make sense to just use "17.3.0.0" as the version number?